date.js

```diff
@@ -1,52 +1,10 @@
-var restaurants = [
-  {
-    id: 1,
-    name: 'local1',
-    category_id: '1',
-    location: {adress: 'adres local1', lat: 51.768766, lng: 19.455707},
-    rate: 4.5,
-  },
+restaurants = getAllRestaurants();
+categories = getAllCategories();
 
-  {
-    id: 2,
-    name: 'local2',
-    category_id: '0',
-    location: {adress: 'ades local2', lat: 51.769897, lng: 19.448322},
-    rate: 4.5,
-  },
+restaurants = restaurants.filter(function(restaurant) {
+  return restaurant.id < 11;
+});
 
-  {
-    id: 3,
-    name: 'local3',
-    category_id: '3',
-    location: {adress: 'adres local3', lat: 51.775697, lng: 19.445487},
-    rate: 2.5,
-  },
-
-  {
-    id: 4,
-    name: 'local4',
-    category_id: '3',
-    location: {adress: 'adres local 4', lat: 52.1, lng: 19.5},
-    rate: 4.5,
-  },
-];
-
-var categories = [
-  {id: 0, name: 'kuchania chinska', count: 10},
-  {id: 1, name: 'kuchania wietnamska', count: 3},
-  {id: 2, name: 'kuchania indyjska', count: 4},
-  {id: 3, name: 'Pizzerie', count: 10},
-  {id: 4, name: 'Kawiarnie', count: 6},
-  {id: 5, name: 'Bary', count: 2},
-];
-
-var date1 = '26-11-2017';
-var date2 = '27-10-2017';
-var date3 = '27-11-2016';
-
-var rates = [
-  {id: 0, username: 'imie nazwisko', rate: 3, created_at: date1, text: 'ok'},
-  {id: 1, username: 'imie1 nazwisko', rate: 3, created_at: date2, text: 'ok'},
-  {id: 2, username: 'imie2 nazwisko', rate: 3, created_at: date3, text: 'ok'},
-];
+categories = categories.filter(function(category) {
+  return category.id < 8;
```

Comment:
let's notify Nikodem about it and bring back displaying _all_ of the sent restaurants